apps/condo/domains/organization/schema/Organization.test.js

```diff
@@ -1,92 +1,230 @@
+                        'name': 'AccessDeniedError',
+                        'path': ['obj'],
+                    })
+                    expect(thrownError.data).toEqual({ 'obj': null })
+                })
+            })
+
+        })
+
+        describe('employed into organization', () => {
+
+            it('cannot without granted "canManageOrganization"', async () => {
+                const admin = await makeLoggedInAdminClient()
+                const [organization] = await createTestOrganization(admin)
+                const [role] = await createTestOrganizationEmployeeRole(admin, organization, {
+                    canManageOrganization: false,
+                })
+                const managerUserClient = await makeClientWithNewRegisteredAndLoggedInUser()
+                await createTestOrganizationEmployee(admin, organization, managerUserClient.user, role)
+
+                let thrownError
+                try {
+                    await updateTestOrganization(managerUserClient, organization.id)
+                } catch (e) {
+                    thrownError = e
+                }
+                expect(thrownError).toBeDefined()
+                expect(thrownError.errors[0]).toMatchObject({
+                    'message': 'You do not have access to this resource',
+                    'name': 'AccessDeniedError',
+                    'path': ['obj'],
+                })
+                expect(thrownError.data).toEqual({ 'obj': null })
+            })
+
+            it('can with granted "canManageOrganization"', async () => {
+                const admin = await makeLoggedInAdminClient()
+                const [organization] = await createTestOrganization(admin)
+                const [role] = await createTestOrganizationEmployeeRole(admin, organization, {
+                    canManageOrganization: true,
+                })
+                const managerUserClient = await makeClientWithNewRegisteredAndLoggedInUser()
+                await createTestOrganizationEmployee(admin, organization, managerUserClient.user, role)
+
+                const [objUpdated, attrs] = await updateTestOrganization(managerUserClient, organization.id)
+                expect(objUpdated.id).toEqual(organization.id)
+                expect(objUpdated.dv).toEqual(1)
+                expect(objUpdated.sender).toEqual(attrs.sender)
+                expect(objUpdated.v).toEqual(2)
+                expect(objUpdated.newId).toEqual(null)
+                expect(objUpdated.deletedAt).toEqual(null)
+                expect(objUpdated.createdBy).toEqual(expect.objectContaining({ id: admin.user.id }))
+                expect(objUpdated.updatedBy).toEqual(expect.objectContaining({ id: managerUserClient.user.id }))
+                expect(objUpdated.createdAt).toMatch(DATETIME_RE)
+                expect(objUpdated.updatedAt).toMatch(DATETIME_RE)
+                expect(objUpdated.updatedAt).not.toEqual(objUpdated.createdAt)
```

Comment:
What about other fields?

Reply:
Added checks for scalars

Reply:
statusTransitions?

Reply:
Added in `15d7fd25`
Skipped in `d53838e1` test for `defaultEmployeeRoleStatusTransitions`, — don't understand yet, why it passes.

Reply:
Fixed in `3c180d60` for `defaultEmployeeRoleStatusTransitions`